ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -643,27 +643,39 @@ def test_matmul(*, x, y, test_flags, backend_fw, fn_name, on_device):
 
 # matrix_norm
 @handle_test(
+    ground_truth_backend="torch",
     fn_tree="functional.ivy.matrix_norm",
-    # ground_truth_backend="numpy",
     dtype_value_axis=helpers.dtype_values_axis(
         available_dtypes=helpers.get_dtypes("float"),
         min_num_dims=2,
         valid_axis=True,
         min_axes_size=2,
         max_axes_size=2,
+        max_value=1e4,
+        min_value=-1e4,
+        abs_smallest_val=1e-4,
         force_tuple_axis=True,
         allow_neg_axes=False,
+        safety_factor_scale="log",
```

Comment:
as `min_value`, `max_value`, etc. are specified, the `safety_factor` arguments isn't required

Reply:
Fixed 👍